rust/src/applayertemplate/logger.rs

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2018 Open Information Security Foundation
+/* Copyright (C) 2021 Open Information Security Foundation
```

Comment:
Generally the model we follow would have this be 2018-2021.  The idea is that the first year is the year of creation, and the second date is the year of the last modification, in a sense to extend the copyright out from the most recent modification.

While legally the year specified is doesn't matter at all, I think we should remain consistent for what its worth.

Reply:
I see, thank you, I wondered why some had two years while others had just one. I'll fix that.